index.html

```diff
@@ -1,232 +1,395 @@
+            </ul>
+            <div>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div>
+              <ul class="languages-works-one">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <div>
+              <button type="button" class="button">
+                <span>See Project</span>
+              </button>
+            </div>
+          </div>
         </div>
+        <div class="works-two">
+          <div class="works-top-two">
+            <a href="#" class="snapshoot-two"
+              ><img src="./images/SnapshootPortfolio2.svg" alt="portfolio-two"
+            /></a>
+          </div>
+          <div class="project-two-info">
+            <h3>Multi-Post Stories</h3>
+            <ul class="project-details-two">
+              <li>Canopy</li>
+              <li>BackEnd Dev</li>
+              <li>2015</li>
+            </ul>
+            <div>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div>
+              <ul class="languages-works-two">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button type="button" class="button">
+              <span>See Project</span>
+            </button>
+          </div>
         </div>
-        </li>
-        <li>
-          <div class="skill-projects-item-one">
-          <p>Frameworks<span><img src="./images/arrow.png" alt="frames-icon"></span></p>
+        <div class="works-three">
+          <div class="works-top-three">
+            <a href="#" class="snapshoot-three"
+              ><img
+                src="./images/SnapshootPortfolio1.svg"
+                alt="portfolio-three"
+            /></a>
           </div>
-          </li>
-          <li>
-          <div class="skill-projects-item-two">
-          <p>Skills<span><img src="./images/arrow.png" alt="skills-icon"></span></p>
+          <div class="project-three-info">
+            <h3>Tonic</h3>
+            <ul class="project-details-three">
+              <li>Canopy</li>
+              <li>BackEnd Dev</li>
+              <li>2015</li>
+            </ul>
+            <div>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div>
+              <ul class="languages-works-three">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button type="button" class="button">
+              <span>See Project</span>
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly improve your styling on **ALL** the `see project` buttons to match the Figma design below:


**figma**
![sp](https://user-images.githubusercontent.com/59397159/172722712-c7d67a45-c97f-4998-b456-941cce4bd49b.png)


**your button**
![btwo](https://user-images.githubusercontent.com/59397159/172722794-76d0c879-c490-430a-9480-5d5533929d32.png)